Description: Update path to aliases file on Debian
Author: Ryan Kavanagh <rak@debian.org>
Origin: vendor
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/smtpd/smtpd.conf
===================================================================
--- opensmtpd.orig/smtpd/smtpd.conf	2014-02-01 15:01:49.516281154 -0500
+++ opensmtpd/smtpd/smtpd.conf	2014-02-01 17:30:00.611211520 -0500
@@ -5,7 +5,7 @@
 listen on localhost
 
 # If you edit the file, you have to run "smtpctl update table aliases"
-table aliases file:/etc/mail/aliases
+table aliases file:/etc/aliases
 
 # Uncomment the following to accept external mail for domain "example.org"
 #accept from any for domain "example.org" alias <aliases> deliver to mbox
